common/vm_wait_primary_nic_mac.yml

```diff
@@ -1,7 +1,7 @@
 # Copyright 2021-2023 VMware, Inc.
 # SPDX-License-Identifier: BSD-2-Clause
 ---
-# Sometime we need to wait for a while to get mac address after create VM
+# Wait for VM's primary network adapter having MAC address
```

Comment:
Add a blank line with '#' in front of it, which is used to separate the task file description and tasks

Reply:
Thanks. Done